packages/account-v2/src/containers/TradingAssessmentForm/TradingAssessmentForm.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
```

Comment:
should we move this to modules since real account creation also needs trading assessment form

Reply:
will move it to modules